src/commons/utils/corio_utils.py

```diff
@@ -327,6 +327,18 @@ def get_report_file_path(corio_start_time) -> str:
     return os.path.join(REPORTS_DIR,
                         f"corio_summary_{corio_start_time.strftime('%Y_%m_%d_%H_%M_%S')}.report")
 
+def get_test_file_path(corio_start_time, test_id) -> str:
+    """
+    Returns test log file path
+
+    :param corio_start_time: Start time for main process.
+    """
+    test_list = []
+    dir_path = LOG_DIR + '/latest/'
```

Comment:
use log path from coinstant.py as already added so it would be consistent if someone change it in that file.

Reply:
replaced with log path in constant.py